website/views/shared/tasks/edit/advanced_options.jade

```diff
@@ -1,69 +1,69 @@
-div(ng-if='::task.type!="reward"')
+div(ng-if='::task._edit.type!="reward"')
```

Comment:
As the type cannot change, there's no need to specify the `_edit` portion. Let's leave it as `task.type`